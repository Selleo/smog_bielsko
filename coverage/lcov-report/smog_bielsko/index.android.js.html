<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for smog_bielsko/index.android.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../prettify.css" />
    <link rel="stylesheet" href="../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../index.html">All files</a> / <a href="index.html">smog_bielsko</a> index.android.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>0/55</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/36</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>0/16</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">0% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>0/22</span>
      </div>
    </div>
  </div>
  <div class='status-line low'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
35
36
37
38
39
40
41
42
43
44
45
46
47
48
49</td><td class="line-coverage quiet"><span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-no">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React,<span class="cstat-no" title="statement not covered" > { Component } from 'react';<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
import { AppRegis<span class="cstat-no" title="statement not covered" >try, StyleSheet, Text, V</span>iew, Button, Navigator, DrawerLayoutAndroid } from 'react-native';
&nbsp;
import Inde<span class="cstat-no" title="statement not covered" >x from './scenes/Index'<span class="cstat-no" title="statement not covered" ></span></span>
import DrawerMen<span class="cstat-no" title="statement not covered" >u from './components/nav/DrawerMenu'<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ><span class="fstat-no" title="function not covered" ></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span></span>
&nbsp;
export defaul<span class="cstat-no" title="statement not covered" ><span class="fstat-no" title="function not covered" >t </span>class smog_bielsko <span class="cstat-no" title="statement not covered" >extends Component {</span></span>
  constru<span class="fstat-no" title="function not covered" >ctor() {<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ></span></span></span>
    super();
<span class="cstat-no" title="statement not covered" >    this.routes = [</span>
      { id: 'i<span class="cstat-no" title="statement not covered" >ndex' }</span>
    ]
<span class="cstat-no" title="statement not covered" >  }<span class="fstat-no" title="function not covered" ></span></span>
&nbsp;
  render() {<span class="cstat-no" title="statement not covered" ></span>
    var navigationView = (
<span class="cstat-no" title="statement not covered" >      &lt;DrawerMenu</span>
        closeDrawer={<span class="fstat-no" title="function not covered" >(stationId) </span>=&gt; this.dra<span class="cstat-no" title="statement not covered" >wer.closeDrawer() || this.nav.replace({id: 'index', stationId: stationId})}</span>
        getCurrentStationId={<span class="fstat-no" title="function not covered" >() =&gt; this.currentSt</span>at<span class="cstat-no" title="statement not covered" >ionId}</span>
      /&gt;
    )
<span class="cstat-no" title="statement not covered" >    return (</span>
      &lt;DrawerLayoutAndroid
        drawerWidth={300}
        drawerPosition={DrawerLayoutAndroid.positions.Left}
        drawerLockMode='unlocked'
        ref={<span class="fstat-no" title="function not covered" >(dra</span>wer) =&gt; <span class="cstat-no" title="statement not covered" >{ return this.drawer = drawer</span> }}
        renderNavigationView={<span class="fstat-no" title="function not covered" >() =&gt; navigationView}</span>&gt;<span class="cstat-no" title="statement not covered" ></span>
        &lt;Navigator
          initialRoute={this.routes[0]}
          initialRouteStack={this.routes}
          renderScene={this.navigatorRenderScene.bind(this)}
        /&gt;
      &lt;/DrawerLayoutAndroid&gt;
    )
  }<span class="fstat-no" title="function not covered" ></span>
&nbsp;
  navigatorRenderScene(route, nav) {
<span class="cstat-no" title="statement not covered" >    this.nav =</span> nav;
<span class="cstat-no" title="statement not covered" >    this.currentStationId = route.stati</span>onId;
<span class="cstat-no" title="statement not covered" >    switch (route.id) {</span>
<span class="branch-0 cbranch-no" title="branch not covered" >      case 'index':</span>
<span class="cstat-no" title="statement not covered" >        return (&lt;Index nav={nav} stationId={route.stationId} /&gt;)</span>
    }
  }<span class="cstat-no" title="statement not covered" ><span class="cstat-no" title="statement not covered" ></span></span>
}
&nbsp;
<span class="cstat-no" title="statement not covered" >AppRegistry.registerComponent('smog_bielsko', () =&gt; smog_b<span class="fstat-no" title="function not covered" >ie</span>lsko);<span class="cstat-no" title="statement not covered" ></span></span>
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="http://istanbul-js.org/" target="_blank">istanbul</a> at Tue Mar 07 2017 12:10:31 GMT+0100 (CET)
</div>
</div>
<script src="../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../sorter.js"></script>
</body>
</html>
