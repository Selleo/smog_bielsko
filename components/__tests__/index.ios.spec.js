import 'react-native';
import React                          from 'react';

import { shallow }                    from 'enzyme';
import chai, { expect }               from 'chai';
import dirtyChai                      from 'dirty-chai';

import smog_bielsko                   from '../../index.ios';
import { data }                       from './mock/data'

const originalExpect = global.expect;

chai.use(dirtyChai);

let wrapper, props;

describe('smog_bielsko', () => {

  it('renders correctly', () => {
    wrapper = shallow(<smog_bielsko />);
    originalExpect(wrapper).toMatchSnapshot();
    expect(wrapper).to.exist();
  });

});
