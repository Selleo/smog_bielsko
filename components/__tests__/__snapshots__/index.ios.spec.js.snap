exports[`smog_bielsko renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <View
    style={
        Object {
            "backgroundColor": "white",
            "flex": 1,
            "flexDirection": "column",
            "justifyContent": "space-around",
          }
    }>
    <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
            Object {
                "fontSize": 18,
                "textAlign": "center",
              }
        }>
        Loading
    </Text>
</View>,
  "nodes": Array [
    <View
      style={
            Object {
                  "backgroundColor": "white",
                  "flex": 1,
                  "flexDirection": "column",
                  "justifyContent": "space-around",
                }
      }>
      <Text
            accessible={true}
            allowFontScaling={true}
            ellipsizeMode="tail"
            style={
                  Object {
                        "fontSize": 18,
                        "textAlign": "center",
                      }
            }>
            Loading
      </Text>
</View>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 0,
      "_context": Object {},
      "_currentElement": <SmogBielsko />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": SmogBielsko {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "routes": Array [
          Object {
            "id": "index",
          },
        ],
        "state": Object {
          "isOpen": false,
          "pendingApp": true,
        },
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <View
          style={
                    Object {
                              "backgroundColor": "white",
                              "flex": 1,
                              "flexDirection": "column",
                              "justifyContent": "space-around",
                            }
          }>
          <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                              Object {
                                        "fontSize": 18,
                                        "textAlign": "center",
                                      }
                    }>
                    Loading
          </Text>
</View>,
        "_debugID": 2,
        "_renderedOutput": <View
          style={
                    Object {
                              "backgroundColor": "white",
                              "flex": 1,
                              "flexDirection": "column",
                              "justifyContent": "space-around",
                            }
          }>
          <Text
                    accessible={true}
                    allowFontScaling={true}
                    ellipsizeMode="tail"
                    style={
                              Object {
                                        "fontSize": 18,
                                        "textAlign": "center",
                                      }
                    }>
                    Loading
          </Text>
</View>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <SmogBielsko />,
}
`;
